araig_utils/launch/export.launch

```diff
@@ -0,0 +1,41 @@
+<launch>
+    <arg name="video_path" default="/home/chang/ARAIG/navel/test3/"/>
```

Comment:
remove this default since it does not make sense outside of this very specific case.